### DISCUSSION TITLE
Help to start gnb

### DISCUSSION THREAD

**Discussion by ChiefMeta** [srsran/srsRAN_Project#460] [General Help] [answered]
I recently upgraded to the latest srsRAN_Project version and I couldn't attach the UE anymore. 

Is there any latest documentation that I can follow? 

Please help!
![1](https://github.com/srsran/srsRAN_Project/assets/83452466/d8dd260a-b28e-45ba-a752-7b671da872ab)
![2](https://github.com/srsran/srsRAN_Project/assets/83452466/e9c185ba-d719-4486-a075-e86af25148d7)
![3](https://github.com/srsran/srsRAN_Project/assets/83452466/fbd88dec-6ccc-4e67-90e0-b6976049bbc5)

**Comment by pgawlowicz** [comment]
Was it possible to connect a previous gnb version to the core in your current setup?

**Comment by ChiefMeta** [comment]
![4](https://github.com/srsran/srsRAN_Project/assets/83452466/c0fb9be4-9367-4b03-bbfc-523cbe7949f9)
I ran the previous version 23.5 and i think there is an issue with the core. Please see attached file. I upgraded the core and everything stops working

**Reply by pgawlowicz** [reply, marked as answer]
@chiefmeta 
Could you check if this solution works for you:
https://github.com/open5gs/open5gs/discussions/2782#discussioncomment-8001181
https://github.com/srsran/srsRAN_Project/discussions/388#discussioncomment-8055894
